specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecretsCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Container Apps Secrets": {
+            "$ref": "./examples/ContainerAppsJob_ListSecrets.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Configuration": {
+      "description": "Non versioned Container Apps Job configuration properties that define the mutable settings of a Container apps job",
+      "type": "object",
+      "required": [
+        "replicaTimeout",
+        "crashLoopRetryLimit",
+        "triggerType"
+      ],
+      "properties": {
+        "secrets": {
+          "description": "Collection of secrets used by a Container app",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/Secret"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "triggerType": {
+          "description": "Type of the job - Schedule | Event | Manual",
+          "enum": [
+            "Scheduled",
+            "Event",
+            "Manual"
+          ],
+          "type": "string",
+          "default": "Single",
+          "x-ms-enum": {
+            "name": "TriggerType",
+            "modelAsString": true
+          }
+        },
+        "replicaTimeout": {
+          "description": "Number of seconds from the startTime for each replica after which the system terminate it.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "replicaRetryLimit": {
+          "format": "int32",
+          "description": "Maximum number of replica retries before failing the whole job.",
+          "type": "integer"
+        },
+        "manualTriggerConfig": {
+          "type": "object",
+          "description": "Manual Trigger configurations. Properties replicaCount and parellelism would be set to 1 by default",
+          "properties": {
+            "replicaCount": {
```

Comment:
@vinisoto since replica count is completions in Kubernetes and parallelism is how many parallel jobs to start, I think it is better to put the work count somewhere here or call it minReplicaCompletions or replicaCompletionCount. It should be
replicaCount <= parallelism